OloEngine/src/OloEngine/Networking/Lockstep/LockstepManager.cpp

```
@@ -0,0 +1,171 @@
+#include "OloEnginePCH.h"
+#include "LockstepManager.h"
+#include "OloEngine/Scene/Scene.h"
+#include "OloEngine/Core/Log.h"
+#include "OloEngine/Debug/Profiler.h"
+
+namespace OloEngine
+{
+    LockstepManager::LockstepManager() = default;
+
+    void LockstepManager::SetTickRate(u32 hz)
```

Clang (14.0.6)

[warning] parameter name 'hz' is too short, expected at least 3 characters

(readability-identifier-length)